spec/spec_helper_acceptance.rb

```diff
@@ -35,6 +35,9 @@
     when 'RedHat'
       on hosts, 'puppet module install stahnma-epel'
       apply_manifest_on hosts, 'include epel'
+      if fact('operatingsystemrelease') =~ /^7/
+        shell('yum install -y iptables-services')
```

Comment:
This should be `on hosts` if we want the replset tests to work (because they are multi-host): https://github.com/puppetlabs/puppetlabs-mongodb/blob/master/spec/acceptance/replset_spec.rb